tensorflow/lite/delegates/gpu/common/testing/interpreter_utils.cc

```diff
@@ -50,12 +50,17 @@ absl::Status InterpreterInvokeWithOpResolver(
     return absl::InternalError("Unable to allocate TfLite tensors");
   }
   for (int i = 0; i < inputs.size(); ++i) {
-    DCHECK_EQ(interpreter->tensor(interpreter->inputs()[i])->type,
-              kTfLiteFloat32);
+    if(interpreter->tensor(interpreter->inputs()[i])->type !=  kTfLiteFloat32){
```

Comment:
Could you explain why this change is needed?

Reply:
Hi @terryheo , When I ran the bazel test command, it gave error saying the DCHECK_EQ macro/function was not defined.

Reply:
Oh I see. We might consider using TFLITE_DCHECK_EQ but this looks fine to me. Thanks!

Reply:
Could you apply TF code style?

https://www.tensorflow.org/community/contribute/code_style

Reply:
Sure, will update the formatting.